tests/unit_tests/test_gameconnection.py

```diff
@@ -438,56 +442,119 @@ async def test_handle_action_GameEnded_ends_game(
     game.on_game_end.assert_called_once()
 
 
-async def test_handle_action_OperationComplete(ugame: Game, game_connection: GameConnection, database):
-    """
-        Sends an OperationComplete action to handle action and verifies that
-    the `coop_leaderboard` table is updated accordingly.
-
-    Requires that the map from `game.map_file_path` exists in the database.
-    """
-
+async def test_handle_action_OperationComplete(
+    ugame: Game,
+    game_connection: GameConnection,
+    database,
+):
     ugame.map_file_path = "maps/prothyon16.v0005.zip"
     ugame.validity = ValidityState.COOP_NOT_RANKED
     game_connection.game = ugame
-
-    secondary = 1
     time_taken = "09:08:07.654321"
-    await game_connection.handle_action("OperationComplete", ["1", secondary, time_taken])
+
+    await game_connection.handle_action(
+        "OperationComplete", [True, True, time_taken]
```

Comment:
the args here should be strings as sent by the game. The lua serializer that the game uses is notoriously weird about serializing booleans. We should be able to handle any of the following inputs to this function for booleans:
1. `True`/`False`
2. `"true"`/`"false"`
3. `1`/`0`

Reply:
Ok, I'll try to test that.